src/screens/OtherProfile/index.js

```diff
@@ -205,14 +206,14 @@ const OtherProfile = () => {
     const __renderBlockIcon = () => {
       if (isBlocking)
         return (
-          <View style={styles.buttonFollowing}>
-            <Text style={styles.textButtonFollowing}>Blocked</Text>
+          <View style={styles.buttonFollowing(dynamicColor)}>
+            <Text style={styles.textButtonFollowing(dynamicColor)}>Blocked</Text>
           </View>
         );
 
       return (
-        <View style={{...styles.btnMsg, borderColor: colors.gray1}}>
-          <BlockIcon width={20} height={20} style={{color: colors.gray1}} />
+        <View style={{...styles.btnMsg(dynamicColor), borderColor: COLORS.gray9}}>
+          <BlockIcon width={20} height={20} style={{color: COLORS.gray9}} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `COLORS` constants. This change should improve consistency across the application.
```diff
- <View style={{...styles.btnMsg, borderColor: colors.gray1}}>
-   <BlockIcon width={20} height={20} style={{color: colors.gray1}} />
+ <View style={{...styles.btnMsg(dynamicColor), borderColor: COLORS.gray9}}>
+   <BlockIcon width={20} height={20} style={{color: COLORS.gray9}} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->